config_example.py

```diff
@@ -3,6 +3,7 @@
 # and additional users that need to use the music downloader
 # feature but are not in any chat
 MUSIC_CHATS = ['12345', '67890']
+# MUSIC_USERS should be list of str ("username") or int (ID)
```

Comment:
I meant that you shouldn't used quotted number as example, it can be confusing in some case

so instead of

`MUSIC_CHATS = ['12345', '67890']`

maybe something like this

`MUSIC_CHATS = ["foo_group_username", -100123456789]`

Reply:
fixed in the new pull request https://github.com/konnovdev/cpop_bot/pull/21